pkg/arvo/app/contact-pull-hook.hoon

```diff
@@ -32,14 +33,11 @@
 ++  on-agent  on-agent:def
 ++  on-watch  on-watch:def
 ++  on-leave  on-leave:def
+++  resource-for-update  resource-for-update:con
 ++  on-pull-nack
   |=  [=resource =tang]
   ^-  (quip card _this)
-  :_  this
-  ?~  (get-contact:con entity.resource)  ~
-  =-  [%pass /pl-nack %agent [our.bowl %contact-store] %poke %contact-update -]~
-  !>  ^-  update:store
-  [%remove entity.resource]
+  [~ this]
```

Comment:
Why has this nack behaviour been removed?

Reply:
no longer relevant given implementation details changing